site/book/05-developing-functions/01-functions-specification.md

```diff
@@ -0,0 +1,20 @@
+In order to enable functions to be developed in different toolchains and
+languages and be interoperable and backwards compatible, the kpt project created
+a standard for the inter-process communication between the orchestrator (i.e.
+kpt CLI) and functions. This standard was published as [KRM Configuration
+Functions Specification][spec] and donated to the CNCF as part of the Kubernetes
+SIG-CLI.
+
+Understanding this standard enables you to have a deeper understanding of how
+things work under the hood. It also enables to create your own toolchain for
+function development if you so desire.
```

Comment:
I am wondering if you considered using `specification` instead of `standard` in the above section.